src/Sarif.Multitool/RewriteCommand.cs

```diff
@@ -19,8 +20,9 @@ public static int Run(RewriteOptions rewriteOptions)
                 SarifLog actualLog = MultitoolFileHelpers.ReadSarifFile<SarifLog>(rewriteOptions.InputFilePath);
 
                 OptionallyEmittedData dataToInsert = rewriteOptions.DataToInsert.ToFlags();
+                IDictionary<string, Uri> originalUriBaseIds = ConstructUriBaseIds(rewriteOptions.UriBaseIds);
```

Comment:
>ConstructUriBaseIds [](start = 62, length = 19)

Now that I've seen what this code does, perhaps `ParseUriBaseIdsOption`?

Reply:
I changed it to ConstructUriBaseIdsDictionary, since this clearly expresses the outcome of the parsing.

---
In reply to: [214520447](https://github.com/Microsoft/sarif-sdk/pull/1062#discussion_r214520447) [](ancestors = 214520447)